changelog/allow_casting_from_typetuple_to_typetuple.dd

```diff
@@ -0,0 +1,15 @@
+Casting between compatible tuples
+
+It was not allowed to cast from built-in tuple to built-in tuple even if they are the same type is.
+
+This is now allowed when they are same length and one can be implicit converted to the other.
+
+---
+alias Tuple(T...) = T;
+
+void foo()
+{
+    Tuple!(int, int) tup;
+    cast(Tuple!(int, int)) tup; // allowed
```

Comment:
Please add an example where a type conversion is required, preferably with a tuple containing more than 1 type.